packages/rule-http-cache/src/rule.ts

```diff
@@ -90,15 +90,44 @@ export default class HttpCacheRule implements IRule {
         /** The predefined patterns for file revving.*/
         const predefinedRevvingPatterns: Array<RegExp> = [
             /*
-             * E.g.:
-             * - https://example.com/assets/script.12345.js
-             * - https://example.com/assets/script-2.12345.js
-             * - https://example.com/assets/script_2-12345.js
-             * - https://example.com/assets/icon_meca_1473335663.svg
+             * E.g.: version/timestamp/hash
              *
-             * Live example: https://regex101.com/r/6VduJO/2/
+             * Live example: https://regex101.com/r/6VduJO/36/
              */
-            /\/(\w|-|_)+(\.|-|_)\w+\.\w+$/i
+
+            /*
+             * - https://example.com/assets/jquery-2.1.1.js
+             * - https://example.com/assets/jquery-2.1.1.min.js
+             * - https://example.com/assets/jquery-3.0.0-beta.js
+             * - https://example.com/assets/favicon.123.ico
+             * - https://example.com/wp-content/uploads/fvm/out/header-cb050ccd-1524626949.min.js
+             */
+            /\/[^/]+[._-]v?\d+(\.\d+(\.\d+)?)?[^/]*\.\w+$/i,
+
+            /*
+             * - https://cdn.example.com/jquery.lazy/1.6.5/jquery.lazy.min.js
+             */
+            /\/v?\d+\.\d+\.\d+.*?\//i,
+
+            /*
+             * - https://s3.amazonaws.com/example/wp-content/uploads/2017/07/15084048/jquery.mediaBoxes.dropdown.js
+             */
+            /\/\d+\//,
```

Comment:
I think this is a bit to general and it will wrongly match a lot of online sites where you have URLs such as `.../id/human-redable-name-of-product`.

Reply:
Which urls will it match that it shouldn't?

Reply:
I think it would be fine to assume that a script is versioned any time there is a folder of just numbers.

Reply:
> any time there is a folder of just numbers.

You can have something like a blog, and then: `2018/<some_title>/<some_script_that_is_related_to_the_blog_post>.js`

Reply:
how are we supposed to distinguish between `/2017/07/<some_revved_wordpress_script>.js` and `/2017/07/<some_non_revved_blog_script>.js` ?

Reply:
How common is it to include non-revved scripts/css in a blog directory?

Reply:
should there be a length limit for the numbers?

```js
/\/\d{5,}\//
```

Reply:
> how are we supposed to distinguish between 

We are not, and even if we try that won't be possible without over complicating things, which we shouldn't. The rule allows configurations exactly for cases like this one that are not common conventions or easily distinguishable.

> How common is it to include non-revved scripts/css in a blog directory?

That was just an example, but having `/<digits>/` is pretty common on the web without actually meaning versioning. 

---

As I said before, matching just `/digits/` is to general, so I'm going to merge this PR without this regex.